package.json

```
@@ -18,18 +18,18 @@
     ]
   },
   "dependencies": {
-    "@hapi/b64": "5.x.x",
-    "@hapi/boom": "9.x.x",
-    "@hapi/content": "^5.0.2",
-    "@hapi/hoek": "9.x.x",
-    "@hapi/nigel": "4.x.x"
+    "@hapi/b64": "^6.0.0",
+    "@hapi/boom": "^10.0.0",
+    "@hapi/content": "^6.0.0",
+    "@hapi/hoek": "^10.0.0",
+    "@hapi/nigel": "^4.0.0"
```

https://github.com/hapijs/nigel/pull/32 has now landed and been published.
```suggestion
    "@hapi/nigel": "^5.0.0"
```